tests/psoc6/pwm.py

```diff
@@ -0,0 +1,9 @@
+### PWM
+from machine import PWM
+
+pwm = PWM("P9_0", freq=50, duty_u16=8192, invert=0)
+print(pwm)
```

Comment:
In the future we should add a test to determine  by an ADC pin if the settings are really applied to the pin. That would be a good test for ADC as well as PWM, right ?